internal/agent/hooks/kcrypt.go

```diff
@@ -37,6 +39,21 @@ func (k Kcrypt) Run(c config.Config) error {
 			time.Sleep(10 * time.Second)
 			return nil // do not error out
 		}
+
+		partitionInfo, err := pi.NewPartitionInfoFromFile(pi.DefaultPartitionInfoFile)
```

Comment:
The first time this code runs, there will be no partition info file. I'm thinking this `NewPartitionFromFile` function should create the partition file if it doesn't exist and return and empty `PartitionInfo` object (with no mappings). I'll add a boolean return value to the function to indicate whether the file existed or not. This way the caller can ignore this value if it doesn't care. I'll open a PR on kcrypt to do that.

Reply:
https://github.com/kairos-io/kcrypt/pull/4

Reply:
done